src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,79 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public final class FileHelper {
+    private static final Logger logger = LogManager.getLogger(FileHelper.class.getName());
+
+    private static final String CHARSET = "UTF-8";
+
+    /**
+     * Prevents instantiation as it is a utility class
+     */
+    private FileHelper() {
+    }
+
+    /**
+     * Returns the File representation at the file path
+     */
+    private static Path getFilePath(String fileDirectory, String fileName) {
+        return Paths.get(fileDirectory, fileName);
+    }
+
+    /**
+     * Attempts to create the directory if it does not exist.
+     * @throws IOException When the directory cannot be created
+     */
+    private static void createDirectoryIfNonExistent(String fileDirectory) throws IOException {
+        File directory = new File(fileDirectory);
+        boolean directoryExists = directory.exists() && directory.isDirectory();
+        if (!directoryExists) {
+            if (directory.mkdirs()) {
+                logger.info("Directory created: " + directory.toString());
+            } else {
+                logger.error("Could not create file directory");
+                throw new IOException();
+            }
+        }
+    }
+
+    public static boolean fileExists(String fileDirectory, String fileName) {
+        return Files.exists(getFilePath(fileDirectory, fileName));
+    }
+
+    /**
+     * Loads the contents of the file into a String
+     * @return The String representation of the contents of the file
+     */
+    public static String loadFileContents(String fileDirectory, String fileName) throws IOException {
+        Path filePath = getFilePath(fileDirectory, fileName);
+        logger.info("Load starting: " + filePath.toAbsolutePath());
+        String contents = new String(Files.readAllBytes(filePath), CHARSET);
+        logger.info("Load successful: " + filePath.toAbsolutePath());
+        return contents;
+    }
+
+    /**
+     * Writes a String to the file
+     * @param fileContents The contents of the file in String form, to be written
+     */
+    public static void writeFileContents(String fileDirectory, String fileName, String fileContents)
```

Comment:
I'm not sure about this, but would it be nicer to have `filePath` instead of separating it to `fileDirectory` and `fileName`? We can have use `getFilePath` in the caller. Again, I am not sure about this. Which one do you think is better?